code/datums/station_traits/neutral_traits.dm

```diff
@@ -200,8 +200,11 @@
 	blacklist = list(/datum/station_trait/announcement_intern, /datum/station_trait/announcement_medbot) //Overiding the annoucer hides the birthday person in the annoucement message.
 	///Variable that stores a reference to the person selected to have their birthday celebrated.
 	var/mob/living/carbon/human/birthday_person
+	///Variable that holds the real name of the birthday person once selected, just incase the birthday person's real_name changes.
+	var/birthday_person_name = ""
 	///Variable that admins can override with a player's ckey in order to set them as the birthday person when the round starts.
 	var/birthday_override_ckey
+	force = TRUE
```

Comment:
While it is statistically likely that often it will be someone's birthday, I assume you did not mean to commit this 😅

Reply:
🎉